src/sync/sync_manager.hpp

```diff
@@ -85,7 +85,10 @@ friend class SyncSession;
     void set_client_should_reconnect_immediately(bool reconnect_immediately);
     bool client_should_reconnect_immediately() const noexcept;
 
-    /// Force sync client to reconnect immediately if the connection was lost.
+    /// Ask all valid sync sessions to perform whatever tasks might be necessary to
```

Comment:
Is this a permanent change? IMO it makes the semantics of this method really muddled, so I would much rather see this getting fixed in Sync (Which is the source of this, right? )